src/EditorFeatures/CSharpTest/MakeRefStruct/MakeRefStructTests.cs

```diff
@@ -61,6 +61,18 @@ struct S
             await TestInRegularAndScriptAsync(text, expected, parseOptions: s_parseOptions);
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeRefStruct)]
+        public async Task FieldInRecordStruct()
+        {
+            var text = CreateTestSource(@"
+record struct S
```

Comment:
:memo: The language doesn't allow `record` and `ref struct` together.

Reply:
interesting. i'm curious why not.

Reply:
in other words, was this an intentional omission, or we just didnt' consider it.

Reply:
We considered it and rejected it.
The main rationale is that ref structs can't be used as type parameter, so `IEquatable<RefStruct>` and generic code we produce for equality comparison isn't possible.

---
In reply to: [609207949](https://github.com/dotnet/roslyn/pull/52464#discussion_r609207949) [](ancestors = 609207949)